pkgs/development/python-modules/term-image/default.nix

```
@@ -35,6 +35,12 @@ buildPythonPackage rec {
     urwid = [ urwid ];
   };
 
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+  ];
```

`pythonRelaxDepsHook` is available by default.

```suggestion
  nativeBuildInputs = [
    pythonRelaxDepsHook
  ];
```

---

Oh, thanks, fixed.

The [manual section](https://nixos.org/manual/nixpkgs/stable/#using-pythonrelaxdepshook) doesn’t mention that and it’s always added to `nativeBuildInputs` in the examples. Should the manual be updated?

---

> The [manual section](https://nixos.org/manual/nixpkgs/stable/#using-pythonrelaxdepshook) doesn’t mention that and it’s always added to nativeBuildInputs in the examples. Should the manual be updated?

See the unstable document.